Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:0.10-onbuild
+# replace this with your application's default port
+EXPOSE 8080 
```

Comment:
One thing to be aware of: this will run your node process *as root*.

Generally, I use the non-`onbuild` tags and set a different `USER` in the Dockerfile. It's more work, but the security gains are well worth it.